### PR TITLE
feat: extend minimal option

### PR DESCRIPTION
**Pull request by cat-master21**
This is almost complete just needs testing and more packages removed. ✨

### CODE REVIEW

**Comment by ajstrongdev** [comment]
Looks good, I am excited to test this

File: minimal-option [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+remmina-plugin-rdp
+remmina-plugin-secret
+remmina-plugin-vnc
+poppler-data
+poppler-utils
+apport-gtk
+apport
+apport-symptoms
+baobab
+dbus-x11
```
Comment (ajstrongdev) [inline minimal-option]: dbus x11 should remain installed, unless it is installed when installing Xorg?

**Comment by cat-master21** [comment]
Currently the makefile actually skips right over it oddly. See the logs of latest build if you want.

File: minimal-option (hunk 1 above)
Comment (cat-master21) [reply minimal-option]: This is not needed as dbus-user-session is all xorg needs plus it gets installed if needed later: https://packages.ubuntu.com/source/kinetic/dbus

**Comment by cat-master21** [comment]
Because apparently `makefile` does not support variables I will have it run a script in /build/minimal instead.

File: minimal-option (hunk 1 above)
Comment (cat-master21) [reply minimal-option]: But yeah this could be removed.

**Comment by ajstrongdev** [comment]
Let me know the progress of its development ASAP, thank you.

**Comment by cat-master21** [comment]
Currently, you can create a image with the option but, it fails to remove Snapd with Snapd's uninstall script failing. Also dpkg complains of broken packages but doesn't tell me what. I am investigating this. Because of this the installer never finishes.

**Comment by ajstrongdev** [comment]
I see. That is concerning.

**Pull request merged**